lib/graphql-ast.explorer.ts

```diff
@@ -46,6 +46,24 @@ export interface DefinitionsGeneratorOptions {
    * @default false
    */
   skipResolverArgs?: boolean;
+
+  /**
+   * If provided, specifies a default generated TypeScript type for custom scalars.
+   * @default 'any'
+   */
+  defaultScalarType?: string;
+
+  /**
+   * If provided, specifies a mapping of types to use for custom scalars
+   * @default undefined
+   */
+  customScalarTypeMapping?: Record<string, string>;
+
+  /**
+   * If provided, specifies a custom header to add to the output file (eg. for custom type imports)
+   * @default undefined
+   */
+  header?: string;
```

Comment:
The `header` name may be somewhat ambigious. Maybe we should use `importStatements` instead(?)

Reply:
Good feedback! I might avoid using the term "import" because, while that's what I'm using it for here, it's really just an open-ended string that gets appended to the default header, and before the rest of the document - could be comments or other content, not necessarily an import statement. I'm open to suggestions!

Reply:
Thanks for the clarification! In this case, we might need to emphasize somehow that this header will not **override** the existing one but will be appended to the default one. With the "header" configuration property name, I'd rather think (as a developer) that I can override the default one. Any ideas?

Reply:
`addtionalHeader` or `subHeader` perhaps?

Reply:
I like `additionalHeader`, FWIW.

Reply:
Done